internal/apif/runner.go

```diff
@@ -0,0 +1,181 @@
+package apif
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/rs/zerolog/log"
+	"github.com/saucelabs/saucectl/internal/apitesting"
+	"github.com/saucelabs/saucectl/internal/job"
+	"github.com/saucelabs/saucectl/internal/region"
+	"github.com/saucelabs/saucectl/internal/report"
+	"github.com/saucelabs/saucectl/internal/tunnel"
+)
+
+// ApifRunner represents an executor for api tests
+type ApifRunner struct {
```

Comment:
Odd, I'm surprised the linter didn't complain 🤔 Just redundant due to `apif.ApifRunner` vs `apif.Runner`.

```suggestion
type Runner struct {
```